changelog.md

```diff
@@ -1,7 +1,9 @@
 # CHANGELOG
 
 ## Unreleased
-
+* [1630](https://github.com/zeta-chain/node/pull/1630) added password prompts for hotkey and tss keyshare in zetaclient
+  Starting zetaclient now requires two passwords to be input; one for the hotkey and another for the tss key-share.
+* `zetaclientd start` : 2 inputs required from stdin
```

Comment:
Need to be put in the right section

```
* `zetaclientd start` : 2 inputs required from stdin
```
This is for breaking change right? I though you made a change to make it optional?

Reply:
Creating this issue, we should write down the instructions for the observer

https://github.com/zeta-chain/node/issues/1701